hack/tools/generate-yaml/Dockerfile

```diff
@@ -12,29 +12,15 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-################################################################################
-##                            DOWNLOAD KUSTOMIZE                              ##
-################################################################################
-FROM debian:stretch-slim as kustomize
-RUN apt-get --assume-no update && apt-get -y install curl
-ARG KUSTOMIZE_VERSION
-ENV KUSTOMIZE_VERSION 2.0.0
-RUN curl -Lo /kustomize \
-    "https://github.com/kubernetes-sigs/kustomize/releases/download/v${KUSTOMIZE_VERSION}/kustomize_${KUSTOMIZE_VERSION}_linux_amd64" && \
-    chmod 0755 /kustomize
-
-
 ################################################################################
 ##                                MAIN STAGE                                  ##
 ################################################################################
-FROM debian:stretch-slim
+ARG BASE_IMAGE=gcr.io/cluster-api-provider-vsphere/extra/manifests-base:v0.4.0-beta.1
+FROM ${BASE_IMAGE}
 LABEL "maintainer" "Andrew Kutz <akutz@vmware.com>"
 
-# Install python
-RUN apt-get --assume-no update && apt-get -y install curl python
-
-# Copy kustomize from the kustomize build stage.
-COPY --from=kustomize /kustomize /usr/local/bin/kustomize
+# move kustomize to an executable path
+RUN mv /kustomize /usr/local/bin/kustomize
```

Comment:
This step could be avoided if we just do it in the base image instead.

Reply:
@akutz I'll leave it for now since the base images PR was approved. Not exactly a time consuming step. :)